internal/compiler/widgets/fluent/std-widgets.slint

```diff
@@ -338,6 +342,10 @@ export TabImpl := Rectangle {
     horizontal-stretch: 0;
     vertical-stretch: 0;
 
+    accessible-role: tab;
+    accessible-has-focus <=> has-focus;
+    accessible-label <=> title;
```

Comment:
I really like this API and approach.

Implementation wise, will these two-way bindings end up as aliases, where the `accessible_string_property` implementation will end up querying the title property? (that's how it looks like to me, but I'm curious if I read that right)

Reply:
The  accessible-* property should be optimized away, and the `accessible_string_property` should just end up querying the `title` property.  (modulo bugs)

Reply:
I think that is what's happening, yes.

Technically these are all sting properties when querying them from the rust code later, even though they are treated as bool (or popentially other types) in slint.